pkg/converter/convert_unix.go

```diff
@@ -864,13 +868,27 @@ func LayerConvertFunc(opt PackOption) converter.ConvertFunc {
 			return nil, errors.Wrap(err, "pack tar to nydus")
 		}
 
+		copyBufferDone := make(chan error, 1)
 		go func() {
 			defer pw.Close()
 			buffer := bufPool.Get().(*[]byte)
 			defer bufPool.Put(buffer)
-			if _, err := io.CopyBuffer(tw, tr, *buffer); err != nil {
-				pw.CloseWithError(err)
+			go func() {
+				_, err := io.CopyBuffer(tw, tr, *buffer)
```

Comment:
Hi @Zephyrcf, thanks! Can we move `copyBufferDone` out to the goroutine at L872 instead of adding a new goroutine?

Reply:
done.